commit 0de0767753ab7416ca4a5dec5d228e62d61d6f80
Author: Adrian Perez de Castro <aperez@igalia.com>
Date:   Thu Sep 15 01:47:26 2016 +0300

    Build only a Linux tarball
    
    Change-Id: I715c2746e60a62cda6ce5e20a63dc319b9a5505e

diff --git a/build/scripts/dist.gant b/build/scripts/dist.gant
index 3051016..aecfdc8 100644
--- a/build/scripts/dist.gant
+++ b/build/scripts/dist.gant
@@ -187,28 +187,12 @@ def layoutAll(Map args, String home, String out, Paths _paths = null, buildJps =
   }
   patchPropertiesFile(paths.distAll, args + [appendices: ["$home/build/conf/ideaCE.properties"]])
 
-  layoutWin(args, home, paths)
-  layoutMac(args, home, paths)
   layoutUnix(args, home, paths)
 
-  def macAppRoot = productProperties.macAppRoot()
-  def winAppRoot = productProperties.winAppRoot()
   def linuxAppRoot = productProperties.linuxAppRoot()
   def archiveName = productProperties.archiveName()
 
-  List pathsDist = new File("${paths.sandbox}/bundled.win.jdk/jre").exists() ? [paths.distAll, paths.distWin, "${paths.sandbox}/bundled.win.jdk"] : [paths.distAll, paths.distWin]
-  buildWinZip("$paths.artifacts/${archiveName}.win.zip", pathsDist, winAppRoot)
-
-  buildCrossPlatformZip("$paths.artifacts/${archiveName}.zip", "${paths.sandbox}/sandbox-ce", [paths.distAll],
-                        paths.distWin, paths.distUnix, paths.distMac)
-
-  String macZip = "$paths.artifacts/${archiveName}.mac.zip"
-  buildMacZip(macAppRoot, macZip, [paths.distAll], paths.distMac, productProperties.extraMacBins)
-
   buildTarGz(linuxAppRoot, "$paths.artifacts/${archiveName}.tar", [paths.distAll, paths.distUnix], productProperties.extraLinuxBins)
-  if (p("jdk.bundled.linux") != "false") {
-    buildTarGz(linuxAppRoot, "$paths.artifacts/${archiveName}-jdk-bundled.tar", [paths.distAll, paths.distUnix, "${paths.sandbox}/bundled.linux.jdk"], ["jre/jre/bin/*"] + productProperties.extraLinuxBins)
-  }
   return info
 }
 
